src/lib/split-view/split-view-panel/split-view-panel.ts

```diff
@@ -0,0 +1,214 @@
+import { CustomElement, attachShadowTemplate, FoundationProperty, coerceBoolean, coerceNumber } from '@tylertech/forge-core';
+import { tylIconDragVerticalVariant } from '@tylertech/tyler-icons/extended';
+import { tylIconDragHandle } from '@tylertech/tyler-icons/standard';
+
+import { BaseComponent, IBaseComponent } from '../../core/base/base-component';
+import { SplitViewPanelPosition, SPLIT_VIEW_PANEL_CONSTANTS } from './split-view-panel-constants';
+import { SplitViewPanelFoundation } from './split-view-panel-foundation';
+import { SplitViewPanelAdapter } from './split-view-panel-adapter';
+import { SplitViewOrientation } from '../split-view/split-view-constants';
+import { ISplitViewBase } from '../core/split-view-base';
+import { IconComponent, IconRegistry } from '../../icon';
+import { RippleComponent } from '../../ripple';
+
+import template from './split-view-panel.html';
+import styles from './split-view-panel.scss';
+
+export interface ISplitViewPanelComponent extends ISplitViewBase, IBaseComponent {
+  position: SplitViewPanelPosition;
```

Comment:
I'm still not sold on the name "position" for this property. If we can put some more info in the docs around it to make it clear then I think that will help. I was a bit confused when I first started using it, but then it started to make sense once I figured it out.

@nickonometry @MikeMatusz do you guys have any thoughts on this one? Sam and I chatted about it, and from his perspective it relates to 1) which side of the panel the handle goes on (which panel the handle is controlling) and 2) which direction the panel will open/close toward when the `open` property is used. We are struggling to come up with a short but descriptive name since those two things negate themselves.

My thought was that we could pick the primary thing it should represent, but I want it to be easy to grok for a new developer using the component is all.

Reply:
I really like the `resizable` concept that came out of the Slack convo for this